vllm/transformers_utils/config.py

```diff
@@ -350,7 +504,7 @@ def get_config(
     trust_remote_code: bool,
     revision: Optional[str] = None,
     code_revision: Optional[str] = None,
-    config_format: ConfigFormat = ConfigFormat.AUTO,
+    config_format: Union[ConfigFormat, str] = ConfigFormat.AUTO,
```

Comment:
If we're going to do this, could we change `ConfigFormat` to a `Literal`? It'd mean we don't have to worry about if we need to add `.value` anywhere